.lektorproject

```diff
@@ -1,5 +1,7 @@
 [project]
 name = creativecommons.github.io
+url_style = absolute
+url = http://creativecommons.github.io/
```

Comment:
how does this work with local development? is there a way to test the feed locally?

Reply:
For testing, I deployed a copy of the site on GitHub pages. https://makkoncept.github.io/testing/feed.xml.
And also while running the site locally, we have to comment out these lines.

Reply:
Having to comment out lines to work locally is not a sustainable solution. I recommend having separate URLs for local and production deployments, keep the local one as the default and update the deployment command in the Travis file to use the production URL.

Reply:
@kgodey In [this feature request](https://github.com/lektor/lektor/issues/377), this issue is discussed ( in "Example Use Case" ). The problem occurs when `url_style=external`. 

I previously assumed that it might not work and tested locally by changing the url to `http://localhost:5000`. I didn't realize that `lektor server` starts the server at localhost:5000 and since the `url_style=absolute`, it works properly. You can check it by running the local server without changing the project config and visiting http://localhost:5000/blog/feed.xml

Reply:
Looks good 👍